x-pack/platform/plugins/shared/stack_connectors/common/openai/schema.ts

```diff
@@ -32,6 +32,27 @@ export const ConfigSchema = schema.oneOf([
     apiProvider: schema.oneOf([schema.literal(OpenAiProviderType.Other)]),
     apiUrl: schema.string(),
     defaultModel: schema.string(),
+     certificateFile: schema.maybe(
+      schema.oneOf([
+        schema.string({ minLength: 1 }),
+        schema.arrayOf(schema.string({ minLength: 1 }), { minSize: 1 }),
+      ])
+    ),
+    certificateData: schema.maybe(schema.string({ minLength: 1 })),
+    privateKeyFile: schema.maybe(
+      schema.oneOf([
+        schema.string({ minLength: 1 }),
+        schema.arrayOf(schema.string({ minLength: 1 }), { minSize: 1 }),
+      ])
+    ),
+    privateKeyData: schema.maybe(schema.string({ minLength: 1 })),
+    verificationMode: schema.maybe(
+      schema.oneOf([
+        schema.literal('full'),
+        schema.literal('certificate'),
+        schema.literal('none'),
+      ], { defaultValue: 'none' })
```

Comment:
I was wrong here, sorry about that!
```suggestion
      ], { defaultValue: 'full' })
```

From @pmuellr 

> Kibana security will eventually notice if the default is none :slightly_smiling_face: (they have in the past for some of our tests).  verificationMode should default to full, otherwise it's a considered a security issue.